test-resources-control-panel/src/main/resources/views/docker/detail.hbs

```diff
@@ -0,0 +1,74 @@
+{{#with controlPanel.body }}
+
+    <div class="card card-light">
+        <div class="card-header">
+            <h3 class="card-title"><code>{{@key}}</code></h3>
+        </div>
+        <div class="card-body">
+            <h2>Managed containers</h2>
+            {{#neq inProgress 0}}
+                There are {{inProgress.size}} containers currently being pulled or starting:
+                <ul>
+                    {{#each pullingContainers as |image|}}
+                        <li><code>{{image}}</code> (pulling)</li>
+                    {{/each}}
+                    {{#each startingContainers as |image|}}
+                        <li><code>{{image}}</code> (starting)</li>
+                    {{/each}}
+                </ul>
+            {{/neq}}
+            <p>The following list only includes the containers whose lifecycle is handled by this Micronaut Test Resources server.</p>
+            <table class="table">
+                <thead>
+                <tr>
+                    <th>Scope</th>
+                    <th>Container name</th>
+                    <th>Image Name</th>
+                    <th>Container id</th>
+                    <th>Network</th>
+                </tr>
+                </thead>
+                <tbody>
+                {{#each managedContainers as |container|}}
+                    <tr>
+                        <td><code>{{container.scope}}</code></td>
+                        <td><code>{{container.name}}</code></td>
+                        <td><code>{{container.imageName}}</code></td>
+                        <td><code>{{container.id}}</code></td>
+                        <td><code>{{container.network}}</code></td>
+                    </tr>
+
+                {{/each}}
+                </tbody>
+            </table>
```

Comment:
You could enclose this part with a condition of having at least one `container` because when it's empty looks weird.